server/game/cards/04.1-FJ/Confession.js

```diff
@@ -0,0 +1,65 @@
+const GameActions = require('../../GameActions/index.js');
+const SpellCard = require('../../spellcard.js');
+
+class Confession extends SpellCard {
+    setupCardAbilities(ability) {
+        this.spellAction({
+            title: 'Noon: Miracle 6',
```

Comment:
```suggestion
            title: 'Noon: Add bounty to a dude',
```
I would use more descriptive action titles.